### PR TITLE
WIP: systemd and autotools experimentation

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1474,17 +1476,30 @@ AC_ARG_WITH(systemd-service, AS_HELP_STRING([--with-systemd-service=PATH],
...
     fi
+]
+,
+[
+    AC_PATH_PROG([systemctl], [systemctl], [no])
+    AS_IF([test "x$systemctl" == "xno"],
+    [AC_MSG_CHECKING([Not a systemd system])],
+    [
+        SYSTEMD_SERVICE_PATH=/usr/lib/systemd/system
+    ])
```
Comment (vpodzime) [inline configure.ac]: I would indent the *then* and *else* parts of the `AS_IF` a bit more so that it is clear they belong to the `AS_IF`.

**Comment by vpodzime** [comment]
@cf-bottom let's see what jenkins has to say

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2306/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2306/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2306/

**Comment by olehermanse** [comment]
@vpodzime I don't know exactly what's going on in those jenkins builds. (Regarding the target not found errors we looked at).

> Looks good to me otherwise. We also need a commit removing the profile.d file from enterprise, right?

I was unsure whether it was necessary to do it at the same time or not. But yes, I will definitely remove it once merged.

**Comment by olehermanse** [comment]
To take this one step (commit) at a time, I created a separate PR here:
https://github.com/cfengine/core/pull/3749

**Comment by olehermanse** [comment]
Closing, for now.

**Pull request closed without merging**